google/cloud/functions/framework.h

```diff
@@ -15,20 +15,59 @@
 #ifndef FUNCTIONS_FRAMEWORK_CPP_GOOGLE_CLOUD_FUNCTIONS_FRAMEWORK_H
 #define FUNCTIONS_FRAMEWORK_CPP_GOOGLE_CLOUD_FUNCTIONS_FRAMEWORK_H
 
+#include "google/cloud/functions/function.h"
 #include "google/cloud/functions/user_functions.h"
 #include "google/cloud/functions/version.h"
 #include <functional>
 
 namespace google::cloud::functions {
 FUNCTIONS_FRAMEWORK_CPP_INLINE_NAMESPACE_BEGIN
 
+/**
+ * Runs function wrapped by @p handler.
+ *
+ * Starts a HTTP server at the address and listening endpoint described by
+ * @p argv, invoking @p handler to handle any HTTP request.
+ *
+ * If @p handler wraps a function with the Cloud Event signature, then the
+ * incoming HTTP requests **MUST** conform to the Cloud Events [HTTP protocol
+ * binding specification][cloud-events-spec].
+ *
+ * @note
+ * When deploying code to Google Cloud Functions applications should **not** use
+ * this function directly. The buildpack will automatically create a `main()`
+ * and invoke `Run()` with the correct parameters. We recommend that application
+ * developers use this function only for local development and integration
```

Comment:
Here and 3 lines up where it advises not to "_use_ this function". I think it would be clearer to say [to not] "_call_ this function".